### ISSUE TITLE
Hide secret in log

### ISSUE THREAD

**Issue by laptevn** [Bardavon-Health/actions-aws-ssm-params-to-env#8]
What is the point of asking if you need to decrypt the value if you provide it to GitHub Actions as non-secret?
This secret will be printed in log in plain text in the next step.

You can add `echo "::add-mask::$MY_VARNAME"` line after you set environment variable in GitHub Actions and it will hide the secret from logs.
This is what I can do myself after your step but the secret will be printed to log once because it was initialized in your step before mine.

**Comment by laptevn** [comment]
This is workaround that prints a secret in log once
```
- name: Mask db username
   run: echo "::add-mask::$TF_VAR_db_username"
- name: Get db username parameter
   uses: Bardavon-Health/actions-aws-ssm-params-to-env@v1.2
   with:
     ssm-path: /blabla/db_username
     prefix: TF_VAR_
     decryption: true
```

If you add add-mask in your step, a secret won't be printed ever

**Comment by Joon** [comment]
Thank you for this @dennybiasiolli , I just used your action. Saved me a ton of time!

**Comment by dennybiasiolli** [comment]
@Joon they merged my fork in the main branch and released a v1.3 🥳

You can switch back to this official version with

```yml
- uses: Bardavon-Health/actions-aws-ssm-params-to-env@v1.3
```
